Resources/Prototypes/Entities/Clothing/Hands/base_clothinghands.yml

```diff
@@ -17,5 +17,6 @@
         - ReagentId: Fiber
           Quantity: 10
   - type: Tag
-    tags: 
-      - ClothMade
+    tags:
+    - WhitelistChameleon
```

Comment:
dont add it to a base prototype

Reply:
`base_clothing.yml` has the tag, it has to be on base items otherwise nothing would work,

Reply:
What should I do instead of adding it to the base prototype?